src/web-ui/src/services/api/authentication-api.service.ts

```diff
@@ -28,6 +28,14 @@ export class AuthenticationApiService {
     return this.http.post<void>(`${this.AUTH_URL}/password-reset`, login);
   }
 
+  public isTokenValid(token: string) {
+    return this.http.get<boolean>(`${this.AUTH_URL}/reset-password/token/${token}/valid`);
```

Comment:
A cóż to jest? Nie było czegoś takiego w planach, chyba że tak chcesz. 😄

Reply:
Znaczy spoko, możesz na front-cie sprawdzać, dzięki temu dasz jakiś user friendly komunikat.

Pierwotnie myślałem że stronkę zawsze pokażesz, ale wysyłając nowe hasło, backend Ci po prostu nie przyjmie przestarzałego/niepoprawnego tokenu.

Ale tak jak mówię, może być, dzięki temu szybciej się dowiesz czy w ogóle jest sens pokazywać formularz. 😃

Reply:
A no i to nie będzie Boolean na zwrotce. Wszystko u nas musi zwracać JSONy. Czyli w zwrotce pewnie będzie coś w stylu `{"valid": true}`